Makefile

```
@@ -22,6 +22,14 @@ docker-run: build-whl ## Runs local Airflow for testing
 		cd dev && astro dev restart; \
 	fi
 
+.PHONY: astro-login
+astro-login: # Login to Astro cloud
+	cd dev && astro login
```

```suggestion
	cd dev && astro login cloud.astronomer-stage.io
```
Probably better to login to stage for dev deployments?